examples_utils/paperspace_dataset_load_check/dataset_upload_checker.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2023 Graphcore Ltd. All rights reserved.
+from typing import NamedTuple, Optional, List
```

Comment:
```suggestion
# Copyright (c) 2023 Graphcore Ltd. All rights reserved.
"""
Description of module usage as you put in the description of the PR 😉
"""
from typing import NamedTuple, Optional, List
```

Reply:
Checks if files in local folder match what is expected from the gradient_dataset_metadata.json

 To use import the check_files_match_metadata function:
 check_files_match_metadata(dataset_folder: str, compare_hash: bool)